app/src/main/java/de/stephanlindauer/criticalmaps/utils/TimeToWordStringConverter.java

```diff
@@ -13,6 +14,8 @@
     private static final int HOUR_MILLIS = 60 * MINUTE_MILLIS;
     private static final int DAY_MILLIS = 24 * HOUR_MILLIS;
 
+    private static final SimpleDateFormat SHORT_DATE_FORMAT = new SimpleDateFormat("dd. MMM", Locale.US);
```

Comment:
This is not _really_ a constant. So maybe just rename it to shortDateFormat, or just inline it since it's only used once anyway?